Master-OB-OpenAPI.json

```diff
@@ -10366,6 +10387,81 @@
             }
           }
         ]
+      },
+      "AlternativeIdentifier": {
+        "type": "object",
+        "description": "Additional identifier for a given record",
+        "properties": {
+          "SourceType": {
+            "$ref": "#/components/schemas/SourceType"
+          },
+          "SourceName": {
+            "$ref": "#/components/schemas/SourceName"
+          },
+          "Identifier": {
+            "$ref": "#/components/schemas/Identifier"
+          },
+          "Description": {
+            "$ref": "#/components/schemas/Description"
+          }
+        }
+      },
+      "AlternativeIdentifiers": {
+        "type": "array",
+        "items": {
+          "$ref": "#/components/schemas/AlternativeIdentifier"
+        }
+      },
+      "Identifier": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "A unique identifier for a record that ties to a data source.",
+            "x-ob-item-type": "stdi:stringItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "",
+            "x-ob-sample-value": {
+              "Value": "be3827bd-0b57-4990-a757-7ee13348356f"
+            }
+          }
+        ]
+      },
+      "SourceType": {
```

Comment:
I don't agree with setting up an enumerated list of data sources. We'll never get it populated, so many users will use "Other" and then have to supply "SourceName". Seems easier just to offer "SourceName" and forgo "SourceType"